projects/coordinator/src/main/java/org/batfish/coordinator/resources/ContainerResource.java

```diff
@@ -0,0 +1,110 @@
+package org.batfish.coordinator.resources;
+
+import java.nio.file.Files;
+import java.util.SortedSet;
+import javax.ws.rs.DELETE;
+import javax.ws.rs.ForbiddenException;
+import javax.ws.rs.GET;
+import javax.ws.rs.NotFoundException;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.Response.Status;
+import javax.ws.rs.core.UriBuilder;
+import javax.ws.rs.core.UriInfo;
+import org.batfish.common.BatfishLogger;
+import org.batfish.common.Container;
+import org.batfish.common.CoordConsts;
+import org.batfish.coordinator.Main;
+
+/**
+ * The ContainerResource is a RESTful service for servicing client API calls at container level.
```

Comment:
Javadoc: anything that is a class should be in `{@link }` tags. (Maybe give the [style guide](https://google.github.io/styleguide/javaguide.html) a quick read).

This resource is not a service -- only the WorkMgrServiceV2 is a RESTful service. This is just a resource.